Documentation/ITK5MigrationGuide.md

```diff
@@ -79,6 +79,7 @@ To modernize your code base, replace:
   * `SimpleFastMutexLock` with `std::mutex`, and `#include "itkSimpleFastMutexLock.h"`  with `#include <mutex>`.
   * `FastMutexLock` with `std::mutex`, and `#include "itkFastMutexLock.h"`  with `#include <mutex>`.
   * `MutexLock` with `std::mutex`, and `#include "itkMutexLock.h"`  with `#include <mutex>`.
+  * `EnableIf<X>::Value` with `std::enable_if<X>::value`, and `#include "itkEnableIf.h"` with `#include <type_traits>`.
```

Comment:
@dzenanz Typo (I guess): `std::enable_if<X>::value` should be `std::enable_if<X>::type`.

Reply:
Good catch. Will be rendered moot by your other suggestion 😄